docs/translating/german.md

```diff
@@ -129,11 +129,7 @@ speakers in the community. It's all about making Zulip friendly and usable.
 
 _"Nachricht" (Facebook, WhatsApp, Transifex)_
 
-- Private Message (PM) - **Private Nachricht (PN)**
-
-Since we try to avoid concatenating words whenever possible, don't use
-"Privatnachricht" . PN is the officially used abbreviation for
-"Private Nachricht" and is used in many German chat forums.
+- Direct Message (DM) - **Direktnachrichten (DN)**
```

Comment:
This looks incorrect since it's using the plural form for the translation instead of the singular form.

```suggestion
- Direct Message (DM) - ** Direktnachricht (DN)**
```

Reply:
@drrosa kindly take a look at the translation in Zulip and guide me.  
![image](https://github.com/zulip/zulip/assets/100958893/2658254b-9962-4cb5-b95e-e6c67ef2d2ad)